include/nbl/asset/utils/IMeshPacker.h

```diff
@@ -239,7 +239,7 @@ class IMeshPacker : public IMeshPackerBase
     //TODO: functions: constructTriangleBatches, convertIdxBufferToTriangles, deinterleaveAndCopyAttribute and deinterleaveAndCopyPerInstanceAttribute
     //will not work with IGPUMeshBuffer as MeshBufferType, move it to new `ICPUMeshPacker`
 
-    TriangleBatches constructTriangleBatches(const MeshBufferType* meshBuffer, IdxBufferParams idxBufferParams, core::aabbox3df* aabbs) const
+    TriangleBatches constructTriangleBatches(const MeshBufferType* meshBuffer, IdxBufferParams idxBufferParams, core::aabbox3df*& aabbs) const
```

Comment:
why a reference to a pointer?

Reply:
otherwise it doesn't work

Reply:
you're changing the pointer?

Reply:
copy of pointer passed to the `commit` function, yes

Reply:
original pointer remains unchanged

Reply:
ok I see the usage and whats going on